packages/docs/content/patterns/Forms/error-validation.mdx

```diff
@@ -0,0 +1,186 @@
+  - Hide the error icon from assistive technology by using `aria-hidden="true"` and provide screen reader text to indicate an error ahead of the error message, typically done with "Error" as the screen reader text.
+- Build your [error message guidance](#error-message-content) alongside the form fields and form content, not as an afterthought.
+- Test to validate that users understand what went wrong and how to fix it.
+
+### Do not
+
+- When using choice children, do not use the error alert. Error messaging should be implemented with the error summary and inline error.
+- Do not use the warning alert if the error needs to be fixed for the user to continue.
+- Do not exceed 6-9 form fields on one page. If many inputs are in an error state, the list might be unmanageable for the user.
+
+## Error message content
+
+An error message, in both inline errors and error summaries, must include what the user can do to fix the error.
+
+- Inline errors only state how to fix the issue so the user completes the field correctly.
+- Summary error messages include introduction text that reminds the user what's needed in order to complete the action. It also lists the fields to fix and their corresponding inline error messages - the exact same content should be used for the field title and inline error message.
+
+### All messages must:
+
+- Be specific, short, and consistent.
+- Be in plain English/plain language.
+- Use imperative style.
+- Specify what the user needs to do to fix the issue (using positive framing).
+- Give enough detail to help the user solve the problem or continue to the next step, while using the fewest words possible.
+  - _Examples:_
+    - "Enter a 10-digit United States phone number."
+    - "Enter an email address with @ symbol and email domain, like ".com"."
+    - "Enter birthdate as MM/DD/YYYY."
+- Use a single error message when there are multiple fields related to the same thing, like date (month, day, year).
+- Use sentence case.
+- End with a period.
+- Include language from the question or fieldset label - this helps match up the error message with the relevant form field.
+  - _Example:_
+    - Field label is "Street address." Inline error message is "Enter a street address with building number and road name." Both the label and error message include "street address" for better user comprehension on how to fix the error.
+- Sound human - read the message out loud to see if it sounds like something you would say.
+- Be tested to validation that users understand what went wrong and how to fix the error.
+
+### Error messages should never:
+
+- Be too generic: General error messages don't help the user to fix the problem.
+  - _Example:_
+    - "An error occurred."
+    - "Answer the question."
+    - "Select an option."
+    - "Required field."
+- Use the verbs "see and "view" unless it would be awkward to use another verb. These can be used, but they should be the last choice (accessibility issue).
```

Comment:
```suggestion
- Use the verbs "see" and "view" unless it would be awkward to use another verb. These can be used, but they should be the last choice (accessibility issue).
```